### PR TITLE
Fix change request bugs

### PR DESCRIPTION
**Pull request by lewisrenfrew**
- changes typeahead to <Search / > (with Andrew's blessing)
- fix some bugs I noticed when using this form over and over again to make change requests for bom ut testing

### CODE REVIEW

File: src/Service.Host/client/src/components/ChangeRequests/ChangeRequest.js [hunk 1]
```diff
@@ -21,19 +21,26 @@ import BomChangesTab from './Tabs/BomChangesTab';
...
+import { changeRequest, changeRequestStatusChange } from '../../itemTypes';
 
 function ChangeRequest() {
     const { id } = useParams();
 
     const reduxDispatch = useDispatch();
 
-    const [item, loading] = useInitialise(() => changeRequestActions.fetch(id), 'changeRequest');
+    const [item, loading] = useInitialise(() => changeRequestActions.fetch(id), changeRequest.item);
+    const statusChangeLoadingLoading = useSelector(reduxState =>
```
Comment (richardiphillips) [inline src/Service.Host/client/src/components/ChangeRequests/ChangeRequest.js:32]: is LoadingLoading deliberate?

**Review by richardiphillips — APPROVED** [review]

File: src/Service.Host/client/src/components/ChangeRequests/ChangeTypes/AssemblyChange.js [hunk 2]
```diff
@@ -26,27 +26,28 @@ function AssemblyChange({ item, creating, handleFieldChange }) {
...
             {creating ? (
                 <Grid item xs={12}>
-                    <Typeahead
+                    <Search
+                        propertyName="newPartNumber"
                         label="Part"
-                        title="Search for a part"
-                        onSelect={handlePartChange}
-                        items={partsSearchResults}
+                        helperText="use Enter to search"
```
Comment (andrewfraser73) [inline src/Service.Host/client/src/components/ChangeRequests/ChangeTypes/AssemblyChange.js:40]: do we need need to mention if tab also works

**Review by andrewfraser73 — APPROVED** [review]
approved not sure if the tab on the search is implemented

File: src/Service.Host/client/src/components/ChangeRequests/ChangeTypes/AssemblyChange.js (hunk 2 above)
Comment (lewisrenfrew) [reply src/Service.Host/client/src/components/ChangeRequests/ChangeTypes/AssemblyChange.js:40]: tab doesn't do anything in this case.... Do we think it should?

**Pull request merged**